recipes/biom-format/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.1.6" %}
+
+package:
+  name: biom-format
+  version: {{ version }}
+
+source:
+  fn: biom-format-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/b/biom-format/biom-format-{{ version }}.tar.gz
+  sha256: 8eefc275a85cc937f6d6f408d91b7b45eae854cd5d1cbda411a3af51f5b49b0d
+
+build:
+  number: 0
+  preserve_egg_dir: True
+  entry_points:
+    - biom=biom.cli:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyqi ==0.3.2  # [py2k]
+    - click
+    - h5py >=2.2.0
+    - numpy >=1.3.0
+    - future >=0.16.0
+    - scipy >=0.13.0
+    - pandas >=0.19.2
+    - six >=1.10.0
+
+  run:
+    - python
+    - click
+    - h5py >=2.2.0
+    - pyqi ==0.3.2  # [py2k]
+    - numpy >=1.3.0
+    - future >=0.15.0
+    - scipy >=0.13.0
+    # missing scipy dependency
+    - libgcc
+    - pandas >=0.19.2
+    - six >=1.10.0
+
+test:
+  imports:
+    - biom
+    - biom.cli
+
+  commands:
+    - biom --help
```

Comment:
I think you need to export C.UTF-8 locale here: https://travis-ci.org/bioconda/bioconda-recipes/jobs/290521631#L974

Reply:
I deliberately didn't set those here since that should be done in the extended base container. It didn't work since there was an error in the Dockerfile, which is now fixed.
Now the job bailed out with an `IOError(28, 'No space left on device')`, so I've taken the recipe out into a separate PR #6400.